docs/getting-started/setup/optimizing-build-size.md

```diff
@@ -0,0 +1,435 @@
+import '@ckeditor/ckeditor5-list/dist/index.css';
+import '@ckeditor/ckeditor5-media-embed/dist/index.css';
+import '@ckeditor/ckeditor5-paragraph/dist/index.css';
+import '@ckeditor/ckeditor5-paste-from-office/dist/index.css';
+import '@ckeditor/ckeditor5-table/dist/index.css';
+import '@ckeditor/ckeditor5-typing/dist/index.css';
+import '@ckeditor/ckeditor5-mention/dist/index.css';
+import '@ckeditor/ckeditor5-case-change/dist/index.css';
+import '@ckeditor/ckeditor5-slash-command/dist/index.css';
+
+ClassicEditor.create( document.querySelector( '#editor' ), {
+	plugins: [
+		Essentials,
+		CKFinderUploadAdapter,
+		Autoformat,
+		Bold,
+		Italic,
+		BlockQuote,
+		CKBox,
+		CKFinder,
+		CloudServices,
+		EasyImage,
+		Heading,
+		Image,
+		ImageCaption,
+		ImageStyle,
+		ImageToolbar,
+		ImageUpload,
+		Indent,
+		Link,
+		List,
+		MediaEmbed,
+		Paragraph,
+		PasteFromOffice,
+		PictureEditing,
+		Table,
+		TableToolbar,
+		TextTransformation,
+		Mention,
+
+		CaseChange,
+		SlashCommand
+	],
+	licenseKey: '<LICENSE_KEY>', // Replace this with your license key.
+	toolbar: {
+		items: [
+			'undo', 'redo',
+			'|', 'heading',
+			'|', 'bold', 'italic',
+			'|', 'link', 'uploadImage', 'insertTable', 'blockQuote', 'mediaEmbed',
+			'|', 'bulletedList', 'numberedList', 'outdent', 'indent', 'caseChange'
+		]
+	},
+	image: {
+		toolbar: [
+			'imageStyle:inline',
+			'imageStyle:block',
+			'imageStyle:side',
+			'|',
+			'toggleImageCaption',
+			'imageTextAlternative'
+		]
+	},
+	table: {
+		contentToolbar: [
+			'tableColumn',
+			'tableRow',
+			'mergeTableCells'
+		]
+	},
+	translations: [
+		clipboardTranslations,
+		coreTranslations,
+		enterTranslations,
+		selectAllTranslations,
+		uiTranslations,
+		undoTranslations,
+		uploadTranslations,
+		widgetTranslations,
+		autoformatTranslations,
+		basicStylesTranslations,
+		blockQuoteTranslations,
+		ckboxTranslations,
+		ckfinderTranslations,
+		headingTranslations,
+		imageTranslations,
+		indentTranslations,
+		linkTranslations,
+		listTranslations,
+		mediaEmbedTranslations,
+		tableTranslations,
+		caseChangeTranslations,
+		slashCommandTranslations,
+	],
+	language: 'pl'
+} );
+```
+</details>
+
+The build size of the project before and after the optimizations is as follows:
+
+|                    	| Before optimization 	| After optimization 	| Improvement 	|
+|--------------------	|---------------------	|--------------------	|-------------	|
+| JavaScript         	| 1,248.02 kB         	| 1,029.01 kB        	| -17.55%     	|
+| CSS                	| 340.45 kB           	| 172.84 kB          	| -49.23%     	|
+| JavaScript gzipped 	| 343.56 kB           	| 288.84 kB          	| -15.93%     	|
+| CSS gzipped        	| 46.16 kB            	| 28.71 kB           	| -37.80%     	|
+
+Thanks to the above optimizations, we were able to reduce the total build size by `386.62 kB` (`72.17 kB` gzipped), which is ~80% of the original size. These results will vary depending on the features you use.
```

Comment:
I can't really admit I get the "386kB out of 1.2MB makes 80%" math.

Reply:
The combined size of JavaScript and CSS has been reduced by 20 to 25% (from `1,248.02 + 340.45` to `1,029.01 + 172.84`).

I think we need to rephrase this sentence to make it clear.

Reply:
```suggestion
Thanks to the above optimizations, we were able to reduce the total build size (JavaScript + CSS) by `386.62 kB` (`72.17 kB` gzipped), which is ~75-80% of the original size. These results will vary depending on the features you use.
```